brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/brevproduksjon/mapper/innvilgelsefp/UndermalInkluderingMapper.java

```diff
@@ -32,7 +34,14 @@ public static boolean skalInkludereInnvilget(Behandling behandling, List<Utbetal
     }
 
     public static boolean skalInkludereAvslag(List<Utbetalingsperiode> utbetalingsperioder, String konsekvens) {
-        return utbetalingsperioder.stream().anyMatch(periode -> !periode.isInnvilget()) && (!KonsekvensForYtelsen.ENDRING_I_BEREGNING.getKode().equals(konsekvens)) ;
+        return utbetalingsperioder.stream()
+                .filter(UndermalInkluderingMapper::ikkePerioderMed4102UtenTapteDager)
+                .anyMatch(Utbetalingsperiode::isAvslått)
+                && (!KonsekvensForYtelsen.ENDRING_I_BEREGNING.getKode().equals(konsekvens)) ;
+    }
+
+    private static boolean ikkePerioderMed4102UtenTapteDager(Utbetalingsperiode periode) {
```

Comment:
Snu denne til periodeMed4102UtenTapteDager? også ta .filter(å -> !periodeMed4102UtenTapteDager)?

metodenavnet nå har "perioder" i seg i stedet for "periode"

Reply:
ja, enig. Har endret